NFT/src/pages/api/files/[filter].ts

```diff
@@ -0,0 +1,22 @@
+import pinataSDK, { PinataMetadataFilter } from "@pinata/sdk";
+import { NextApiRequest, NextApiResponse } from "next";
+
+const pinata = new pinataSDK({ pinataJWTKey: process.env.PINATA_JWT });
+
+export const getNFTMetadata  = async (filter?: PinataMetadataFilter) => {
+  const metadata = filter;
+  // TODO: support pagination for an explore page
+  const nftData = await pinata.pinList({ metadata, status: "pinned" });
```

Comment:
We should add error handling and put this in a try catch in case there is an error on the Pinata end.